h2o-py/h2o/automl/autoh2o.py

```diff
@@ -72,6 +75,11 @@ def __init__(self,
                   "*******************************************************************\n" \
                   "\nVerbose Error Message:")
 
+        
+        #nfolds must not be negative or equal to 1:
+        assert nfolds >= 0, "nfolds cannot be negative. Use nfolds >=2 if you want cross-valiated metrics and Stacked Ensembles or use nfolds = 0 to disable."
+        assert nfolds is not 1, "nfolds = 1 is an invalid value. Use nfolds >=2 if you want cross-valiated metrics and Stacked Ensembles or use nfolds = 0 to disable."       
+
```

Comment:
It seems `nfolds` just needs to be >=2. You can just report back what you got for `nfolds` in the exception so the user knows what was passed in.

Reply:
Nice cleanup of some of the help strings in the @API annotations.  Might as well fix the others so they are also complete sentences.

Also, I feel like "nfolds == 1" should mean "don't use xval; it's only one fold", just like 0 does.  Thoughts?

Reply:
@rpeck Ok I can clean up the rest of the strings.  I think we should be consistent with all the H2O algos, which is that `nfolds == 1` is invalid.  Agreed that it would have made more sense in the beginning to use `nfolds = 1` instead of `nfolds = 0` as the "no CV" option.

Reply:
Okay, then can check `assert nfolds >=2 or nfolds ==0` ? 

And yes, always good to report back.